### PR TITLE
added dates and styled posts

### PR DESCRIPTION
**Pull request by tobold**
posts now show date created in the format: hour:minute (day/month/year)
posts use the post-stylesheet.css for styling

### CODE REVIEW

**Review by Alexander-Blair — DISMISSED** [review]
Need a test for the date displayed

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,5 +1,18 @@
 <% @posts.each do |post| %>
-  <p><%= post.message %></p>
+  <div class="post">
+    <div class="posthead">
+      <span class="postuser">
+        (username here)
+      </span>
+      <span class="postdate">
+        posted at: <%= post.created_at.strftime("%H:%M (%d/%m/%y)") %>
```
Comment (dearshrewdwit) [inline app/views/posts/index.html.erb]: Nice! Can you move this logic to the model and write a test for it?

File: app/assets/stylesheets/post-stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.post {
+  width:40%;
+  margin:15px;
+}
+
+.posthead {
```
Comment (dearshrewdwit) [inline app/assets/stylesheets/post-stylesheet.css]: can you use/find a good naming convention for your selectors?

**Review by dearshrewdwit — DISMISSED** [review]

File: app/models/post.rb [hunk 3]
```diff
@@ -1,2 +1,5 @@
 class Post < ApplicationRecord
+  def postedat
+    self.created_at.strftime("%H:%M (%d/%m/%y)")
```
Comment (dearshrewdwit) [inline app/models/post.rb]: no need for `self` here

File: app/models/post.rb [hunk 4]
```diff
@@ -1,2 +1,5 @@
 class Post < ApplicationRecord
+  def postedat
```
Comment (dearshrewdwit) [inline app/models/post.rb]: can you rename this to something a tad more readable, perhaps follow `snake_case` 👀

File: spec/features/posts-show-the-date-they-were-posted_spec.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.feature "Date Posted", type: :feature do
+  scenario "Newly created posts show the date they were posted" do
+    visit "/posts"
+    click_link "New post"
+    fill_in "Message", with: "Hello, world!"
+    click_button "Submit"
+    post = Post.find_by(message: "Hello, world!")
```
Comment (dearshrewdwit) [inline spec/features/posts-show-the-date-they-were-posted_spec.rb:9]: for now this is ok. This should be updated when factory girl is ready in master

**Review by dearshrewdwit — DISMISSED** [review]

**Review by chuk-chuk — APPROVED** [review]
great job!

**Pull request merged**